components/developer/nasm/Makefile

```diff
@@ -16,26 +16,38 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		nasm
-COMPONENT_VERSION=	2.11.08
+COMPONENT_VERSION=	2.13.01
+COMPONENT_FMRI=	developer/assembler/nasm
+COMPONENT_SUMMARY=	The Netwide Assembler
+COMPONENT_CLASSIFICATION=	Development/Other Languages
+COMPONENT_PROJECT_URL=	http://www.nasm.us/
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_HASH=	sha256:e76596043d67d31c61321adf4ab8e1768974c1b497b8d4e9adf64d8b83fb19c0
-COMPONENT_PROJECT_URL=	http://www.nasm.us/
-COMPONENT_ARCHIVE_URL=	http://www.nasm.us/pub/nasm/releasebuilds/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH=	\
+  sha256:0cd4327b114e459a84717260fad4238b445d3511cf0341249072204867698e09
+COMPONENT_ARCHIVE_URL= \
+  http://www.nasm.us/pub/nasm/releasebuilds/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE=	BSD
 
 include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/configure.mk
 include $(WS_MAKE_RULES)/ips.mk
 
-IPS_COMPONENT_VERSION=	2.11.8
+IPS_COMPONENT_VERSION=	2.13.1
 
 CFLAGS+="-I$(BUILD_DIR_$(BITS))"
 
-COMPONENT_POST_CONFIGURE_ACTION += (mkdir -p $(BUILD_DIR_32)/lib $(BUILD_DIR_64)/lib)
+CONFIGURE_BINDIR.64 = $(CONFIGURE_BINDIR.32)
+
+COMPONENT_POST_CONFIGURE_ACTION += (mkdir -p $(BUILD_DIR_64)/lib)
 
 COMPONENT_INSTALL_ENV += INSTALLROOT="$(PROTO_DIR)"
 
-build:		$(BUILD_32_and_64)
+build:		$(BUILD_64)
```

Comment:
The format is documented as being selected explicitely and elf == elf32.
I am testing on a rebuild of libjpeg-turbo.